Fabric.Authorization.Client/scripts/Fabric.Authorization.Client.nuspec

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Did we end up needing a nuspec file? I thought I remembered something about .net standard project using the .csproj file to figure out what needs to be packed. I could be wrong though.